modules/elasticsearch/charts.go

```diff
@@ -0,0 +1,581 @@
+package elasticsearch
+
+import (
+	"github.com/netdata/go-orchestrator/module"
+)
+
+type (
+	Charts = module.Charts
+	Dims   = module.Dims
+	Vars   = module.Vars
+)
+
+// TODO: indices operations charts: query_latency, fetch_latency, index_latency, flush_latency
+// if they are needed at all..
+
+var (
+	nodeCharts = Charts{
+		// Indices Indexing
+		{
+			ID:    "node_indices_indexing_operations",
+			Title: "Indexing Operations",
+			Units: "operations/s",
+			Fam:   "indices indexing",
+			Ctx:   "elasticsearch.node_indices_indexing",
+			Dims: Dims{
+				{ID: "node_indices_indexing_index_total", Name: "index", Algo: module.Incremental},
+			},
+		},
+		{
+			ID:    "node_indices_indexing_operations_current",
+			Title: "Indexing Operations Current",
+			Units: "operations",
```

Comment:
Is this missing an `/s`?

Reply:
i add `/s` to rates which are dimensions with `incremental` algorithm, this one is not